<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scalar Field Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePathLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>331</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select the path to the data file.</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>571</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Operation</string>
    </property>
    <widget class="QRadioButton" name="isoRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Compute Iso-contours</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="minimaRadioButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Find Local Minima</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="maximaRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Find Local Maxima</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="saddleRadioButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Find Saddle Points</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Isovalue:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="isovalueLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Process Data</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="resultsTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>571</width>
      <height>291</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
